### ISSUE TITLE
Infinite Ore + EXP Duplication Exploit with Skills

### ISSUE THREAD

**Issue by JavaLikesAgua** [adam9899/Issues#555]
Not sure if this is known about/intended, but if you level up your mining to have the chance to double ore mining, along with getting EXP from mining, you can essentially get one of about any ore, place it in the world, mine it, and continuously exponentially get more of that ore and exponentially more EXP per ore excavate. 

Only works if you invest the levels into the skill tree, but ridiculously broken, especially once you end up with an infinite wand and a full inventory of the ore .... 

I suppose the solution to this would be that these skills only apply to non-player placed blocks?

**Comment by Timeslice42** [comment]
Yeah, this is why the mod drops chunks instead of ores. Having the chunks automatically convert to the ores is just asking for trouble.

**Comment by Avalonzo** [comment]
I already reported this back in February #211 but didn't notice the XP gain til yesterday.
